src/GovUk.Education.ExploreEducationStatistics.Notifier/Functions/PublicationSubscriptionFunctions.cs

```diff
@@ -22,14 +22,14 @@ namespace GovUk.Education.ExploreEducationStatistics.Notifier.Functions;
 
 public class PublicationSubscriptionFunctions(
     ILogger<PublicationSubscriptionFunctions> logger,
-    IOptions<AppSettingsOptions> appSettingsOptions,
+    IOptions<AppOptions> appSettingsOptions,
```

Comment:
Might wanna change the parameter name to `appOptions` also here

Reply:
Yep fair point! Updated and for all the other cases as well